### PR TITLE
docs(apps/code-examples): cleanup CSS class usage

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: apps/code-examples/src/app/code-examples/modals/confirm/confirm-demo.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <button
   aria-haspopup="dialog"
-  class="sky-btn sky-btn-default sky-margin-inline-sm"
+  class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (blackbaud-johnly) [inline apps/code-examples/src/app/code-examples/modals/confirm/confirm-demo.component.html]: Is this change correct? Elsewhere, we change `default` to `lg`, so it seems odd to change `sm` to `lg` unless `sm` was a bug in the first place.

Comment (Blackbaud-SteveBrush) [reply apps/code-examples/src/app/code-examples/modals/confirm/confirm-demo.component.html]: Good catch. I'm not sure how that one got changed!

File: apps/code-examples/src/app/code-examples/colorpicker/colorpicker/programmatic/colorpicker-demo.component.html [hunk 2]
```diff
@@ -9,23 +9,23 @@
 <p>
   <button
     type="button"
-    class="sky-btn sky-btn-primary sky-margin-inline-default"
+    class="sky-btn sky-btn-primary sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/colorpicker/colorpicker/programmatic/colorpicker-demo.component.html]: all these button spacings should use `sky-margin-inline-sm`

File: apps/code-examples/src/app/code-examples/angular-tree-component/angular-tree/advanced/angular-tree-demo.component.html [hunk 3]
```diff
@@ -102,7 +102,7 @@ <h4 id="sky-demo-modes">Modes</h4>
   </form>
   <!-- #endregion -->
 
-  <div class="sky-padding-even-default app-demo-container">
+  <div class="sky-padding-even-md app-demo-container">
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/angular-tree-component/angular-tree/advanced/angular-tree-demo.component.html]: guessing this should use `sky-padding-even-xl` if it is a larger container

File: apps/code-examples/src/app/code-examples/autonumeric/autonumeric/currency/autonumeric-demo.component.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <form [formGroup]="formGroup" novalidate>
   <sky-input-box>
-    <label class="sky-control-label" [for]="autonumericInput.id">
+    <label class="sky-font-data-label" [for]="autonumericInput.id">
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/autonumeric/autonumeric/currency/autonumeric-demo.component.html]: Think we can leave these as `sky-control-label` based on the way input box is built, but would need to verify

File: apps/code-examples/src/app/code-examples/forms/checkbox/basic/checkbox-demo.component.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <form [formGroup]="myForm" (ngSubmit)="onSubmit()">
-  <div class="sky-margin-stacked-default">
+  <div class="sky-margin-stacked-lg">
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/forms/checkbox/basic/checkbox-demo.component.html]: should be -xl

File: apps/code-examples/src/app/code-examples/forms/checkbox/icon-group/checkbox-demo.component.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <form [formGroup]="myForm" (ngSubmit)="onSubmit()">
-  <div class="sky-switch-icon-group sky-margin-stacked-default">
+  <div class="sky-switch-icon-group sky-margin-stacked-lg">
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/forms/checkbox/icon-group/checkbox-demo.component.html]: should be -xl

File: apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html [hunk 7]
```diff
@@ -18,51 +18,31 @@
 
 <p>
   <button
-    class="
-      sky-btn
-      sky-btn-default
-      sky-margin-inline-default
-      sky-margin-stacked-default
-    "
+    class="sky-btn sky-btn-default sky-margin-inline-lg sky-margin-stacked-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html]: not sure why these buttons have stacked margin

File: apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html [hunk 8]
```diff
@@ -18,51 +18,31 @@
 
 <p>
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html]: why are these buttons in a paragraph?

**Review by Blackbaud-ToddRoberts — CHANGES_REQUESTED** [review]

File: apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html (hunk 7 above)
Comment (Blackbaud-SteveBrush) [reply apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html]: Removed.

File: apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html (hunk 8 above)
Comment (Blackbaud-SteveBrush) [reply apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html]: Probably so they'll have margin; I'll add `sky-margin-stacked-xl` to a DIV instead.

File: apps/code-examples/src/app/code-examples/autonumeric/autonumeric/currency/autonumeric-demo.component.html (hunk 4 above)
Comment (Blackbaud-SteveBrush) [reply apps/code-examples/src/app/code-examples/autonumeric/autonumeric/currency/autonumeric-demo.component.html]: I can revert these, but _should_ the input box component work with `.sky-font-data-label` as well as `.sky-control-label`?

File: apps/code-examples/src/app/code-examples/popovers/dropdown/remote/dropdown-demo.component.html [hunk 9]
```diff
@@ -1,46 +1,46 @@
 <p>
   <button
-    class="sky-btn sky-btn-default sky-margin-inline-default"
+    class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/popovers/dropdown/remote/dropdown-demo.component.html]: these should be inline-sm

File: apps/code-examples/src/app/code-examples/datetime/date-range-picker/basic/date-range-picker-demo.component.html [hunk 10]
```diff
@@ -48,47 +48,47 @@ <h3>Form control value</h3>
 <h3>Manipulate value and state</h3>
 
 <button
-  class="sky-btn sky-btn-default sky-margin-inline-default"
+  class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/datetime/date-range-picker/basic/date-range-picker-demo.component.html]: these should be inline-sm

File: apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html [hunk 11]
```diff
@@ -14,57 +14,37 @@
...
-<p>
+<div class="sky-margin-stacked-xl">
   <button
-    class="
-      sky-btn
-      sky-btn-default
-      sky-margin-inline-default
-      sky-margin-stacked-default
-    "
+    class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/indicators/tokens/custom/tokens-demo.component.html]: these should be inline-sm

File: apps/code-examples/src/app/code-examples/indicators/wait/page/wait-demo.component.html [hunk 12]
```diff
@@ -1,14 +1,14 @@
 <button
   type="button"
-  class="sky-btn sky-btn-default sky-margin-inline-default"
+  class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/indicators/wait/page/wait-demo.component.html]: these should be inline-sm

File: apps/code-examples/src/app/code-examples/lists/repeater/add-remove/repeater-demo.component.html [hunk 13]
```diff
@@ -32,10 +32,10 @@
   </sky-repeater>
 </div>
 
-<button class="sky-btn sky-margin-inline-default" (click)="addItem()">
+<button class="sky-btn sky-margin-inline-lg" (click)="addItem()">
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/lists/repeater/add-remove/repeater-demo.component.html]: these should be inline-sm

File: apps/code-examples/src/app/code-examples/popovers/popover/programmatic/popover-demo.component.html [hunk 14]
```diff
@@ -4,7 +4,7 @@
 ></sky-help-inline>
 
 <button
-  class="sky-btn sky-btn-default sky-margin-inline-default"
+  class="sky-btn sky-btn-default sky-margin-inline-lg"
```
Comment (Blackbaud-ToddRoberts) [inline apps/code-examples/src/app/code-examples/popovers/popover/programmatic/popover-demo.component.html]: these should be inline-sm

**Review by Blackbaud-ToddRoberts — CHANGES_REQUESTED** [review]

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Pull request merged**